### DISCUSSION TITLE
navbar size

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6530] [Q&A] [answered]
How to change navbar size, to make it smaller?
![grafik](https://user-images.githubusercontent.com/54640740/151899095-9e8239da-0ffc-46bf-baac-6219252d7784.png)

**Comment by slorber** [comment, marked as answer]
There are CSS variables that you can use

    `--ifm-navbar-height`

Please ask your questions first on Discord
